draft-ietf-tls-external-psk-guidance.md

```diff
@@ -150,17 +153,20 @@ External PSK authentication in TLS allows endpoints to authenticate connections
 using previously established keys. These keys do not provide protection
 of endpoint identities (see {{endpoint-privacy}}), nor do they provide
 non-repudiation (one endpoint in a connection can deny the conversation).
+Protection of endpoint identities and protection against an endpoint denying
+the conversation are possible when a fresh TLS handshake is performed.
```

Comment:
What is the point of this new text?

Reply:
Look at the two sentences that come before this one.  Those sentences say that the PSK does not offer these things, so if you need these things, a fresh handshake is the way to get them.

Reply:
Yes, I get that, but that seems obvious. Does it need to be said explicitly?

Reply:
(I do not feel strongly about this, so let's just land it.)